activesupport/lib/active_support/cache.rb

```diff
@@ -851,6 +855,7 @@ def load(payload)
           elsif payload.start_with?(MARK_61)
             return Marshal.load(payload)
           else
+            # If you add new supported formats, please update #valid_format? too
```

Comment:
This is a little bit messy, but the alternative was copying the implementation of this method into the `MemCacheStore` which felt like a worse idea.